docs/web/interceptors.mdx

```diff
@@ -152,3 +152,30 @@ import { elizaClient } from "./eliza-client.js";
 
 const res = elizaClient.say({ sentence: "Hey!" }, { contextValues: createContextValues().set(kLogBody, true) });
 ```
+
+## Setting `fetch()` options
+
+Another valuable use case for interceptors is customizing the Fetch API for individual requests by leveraging
+the `request.init` object.
+
+For example, by default, Connect sets the Fetch option `redirect` to `error`, which means that a network error will
```

Comment:
Can we link to https://developer.mozilla.org/en-US/docs/Web/API/fetch#redirect ?